functions in JS/index.html

```diff
@@ -0,0 +1,18 @@
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    
+    <script src="main.js"></script>
+
+</head>
+
+<body>
+	<h1>
+		FUNCTIONS IN JS
+	</h1>
```

Comment:
remove all extra lines from the code

Reply:
@t2013anurag Here are you suggesting to remove all the white spaces or should I shorten my code?

Reply:
remove extra new lines., use only one return statement in code.